covalent/_workflow/postprocessing.py

```diff
@@ -100,9 +100,9 @@ def _postprocess(self, *ordered_node_outputs: List[Any]) -> Any:
         """
         with active_lattice_manager.claim(self.lattice):
             self.lattice.post_processing = True
-            app_log.debug(f"Postprocessing lattice with attrs: {self.lattice.__dict__}")
+            # app_log.debug(f"Postprocessing lattice with attrs: {self.lattice.__dict__}")
```

Comment:
@kessler-frost I commented them out so that they can be uncommented later - but at the same time I don't want it to add noise to the logs. Do you prefer I uncomment it or remove it?

Reply:
If you think it'll be useful later then I would suggest let's uncomment it. I don't think removing a couple of lines is going to have any significant impact in the logs, but I see your point about the necessity of it. I would say it's up to you as you've spent more time debugging these parts, so either let's have it or remove it, but not keep it as commented 😅 .

Reply:
Choosing to remove it!